gui.py

```diff
@@ -48,6 +79,11 @@ def configureButtons(self, window):
         editingModes.insert(0, "None")
         self.lastSelectedEditingMode = tkinter.StringVar(window)
 
+        networkModes = ["Scale-Free Only", "Small-World Only", "Both (Color Coded)"]
```

Comment:
Scale-Free and Small-World have been written as Scale Free and Small World (without hyphen) throughout documentation and rest of codebase. Choose one way and make things consistent throughout the project.